.github/ISSUE_TEMPLATE/1.bug_report.yaml

```diff
@@ -109,10 +110,11 @@ body:
     attributes:
       label: Reproduction steps
       description: |
-        How do you trigger this bug? Please walk us through it step by step. 
+        How can this bug be reproduced?
+        As a general guideline, ask yourself: "will running the exact steps below in an empty directory create an Amplify project that demonstrates the bug behavior?"
```

Comment:
Thoughts on removing the _guideline_ part- 


How can this bug be reproduced? Will running the **exact** steps below _in an empty directory_ create an Amplify project that demonstrates the bug behavior?

Reply:
```suggestion
        Will running the **exact** steps below _in an empty directory_ create an Amplify project that demonstrates the bug behavior?"
```